### ISSUE TITLE
 admob.js not found on chrome (ionic serve)

### ISSUE THREAD

**Issue by astroanu** [appfeel/admob-google-cordova#22]
it's obvious the file is not there and i guess admob policies wont allow you to run admob.js on chrome. However is it possible to run admob on both chrome and android but show ads only if the device is a mobile.

The reason is currently i prefer running the app on chrome when testing. It's kinda annoying to remove and re add the admob code every time i test on Android.

**Comment by appfeel** [comment]
Hi @astroanu, I'm not really understanding the problem, normally you don't need to add and remove the code, simply check `if (window.admob) {}` before any call to admob. For ionic there are also wrapper files you can check documentation here: https://github.com/appfeel/admob-google-cordova/wiki/Angular.js,-Ionic-apps

Please tell me if this resolves your issue.

**Comment by astroanu** [comment]
hey, not really, the thing is the code breaks on chrome even before going to execute any. 

````
var app = angular.module('myApp', ['admobModule']); // <----- on chrome js breaks here

    app.config(['admobSvcProvider', function (admobSvcProvider) { // <----- on chrome js breaks here
      admobSvcProvider.setPrefix('myTag~');
    }]);


    app.run(['admobSvc', function (admobSvc) { // <----- on chrome js breaks here
      $rootScope.$on('myTag~' + admob.events.onAdOpened, function onAdOpened(evt, e) {
        console.log('adOpened: type of ad:' + e.adType);
      });
    }]);

````

**Comment by appfeel** [comment]
@astroanu first sorry for the delay and thanks for your contribution by reporting this issue.
Right now it's solved in v4.1.9 and you should have no problems in browser.

**Comment by astroanu** [comment]
thanks. i'll check
